packages/date-picker/src/DatePicker/components/DateInput/DateInput.scss

```diff
@@ -0,0 +1,96 @@
+@import "~@kaizen/design-tokens/sass/color";
+@import "~@kaizen/component-library/styles/animation";
+@import "~@kaizen/deprecated-component-library-helpers/styles/layout";
+@import "~@kaizen/deprecated-component-library-helpers/styles/type";
+@import "~@kaizen/deprecated-component-library-helpers/styles/color";
+@import "~@kaizen/component-library/styles/utils";
+@import "~@kaizen/component-library/styles/mixins/forms";
+@import "../../mixins";
+
+// -----------------------------------------------
+// Date Input
+// -----------------------------------------------
+
+///////////////////////////////////////////////////
+// MIXINS
+///////////////////////////////////////////////////
+
+@mixin input-icon($color, $reversed) {
+  color: $color;
+
+  .withReversed & {
+    color: $reversed;
+  }
+}
+
+// Vars
+$input-disabled-opacity: 0.3;
+
+.iconButton {
+  @include button-reset;
+  display: inherit;
+  padding: 0.25rem $spacing-xs;
+  border: $border-solid-border-width $border-solid-border-style transparent;
+  border-radius: $border-solid-border-radius;
+  position: relative;
+
+  &:hover:not([disabled]) {
+    background-color: $color-blue-100;
+    color: $color-blue-500;
+  }
+
+  &:global(.focus-visible) {
+    outline-offset: -2px;
+    outline: $color-blue-500 $border-focus-ring-border-style
+      $border-focus-ring-border-width;
+  }
+
+  :focus {
+    outline: none;
+  }
+
+  .disabled {
+    opacity: $input-disabled-opacity;
+  }
+
+  &.calendarActive {
+    color: $color-blue-500;
+    background-color: $color-blue-100;
+  }
+
+  svg {
+    margin-top: 2px;
+  }
+}
+
+///////////////////////////////////////////////////
+// VALIDATION STYLES
+///////////////////////////////////////////////////
+
```

Comment:
We don't currently have any stories showcase the success or caution states for the datepicker - is this something we're able to simulate in the stickersheet or a separate story?